src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -133,12 +133,111 @@ const timezoneLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log)
+};
+
+const silentLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.silent')) {
+    log(
+      '"logging.silent" has been deprecated and will be removed ' +
+        'in 8.0. Moving forward, you can use "logging.root.level:off" in your logging configuration. '
+    );
+  }
+  return settings;
+};
+
+const verboseLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.verbose')) {
+    log(
+      '"logging.verbose" has been deprecated and will be removed ' +
+        'in 8.0. Moving forward, you can use "logging.root.level:all" in your logging configuration. '
+    );
+  }
+  return settings;
+};
+
+const jsonLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  // We silence the deprecation warning when running in development mode because
+  // the dev CLI code in src/dev/cli_dev_mode/using_server_process.ts manually
+  // specifies `--logging.json=false`. Since it's executed in a child process, the
+  // ` legacyLoggingConfigSchema` returns `true` for the TTY check on `process.stdout.isTTY`
+  if (has(settings, 'logging.json') && settings.env !== 'development') {
+    log(
+      '"logging.json" has been deprecated and will be removed ' +
+        'in 8.0. To specify log message format moving forward, ' +
+        'you can configure the "appender.layout" property for every custom appender in your logging configuration. ' +
+        'There is currently no default layout for custom appenders and each one must be declared explicitly. ' +
+        'For more details, see ' +
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx.'
+    );
+  }
+  return settings;
+};
+
+const logRotateDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.rotate')) {
+    log(
+      '"logging.rotate" and sub-options have been deprecated and will be removed in 8.0. ' +
+        'Moving forward, you can enable log rotation using the "rolling-file" appender for a logger ' +
+        'in your logging configuration. For more details, see ' +
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx#rolling-file-appender'
+    );
+  }
+  return settings;
+};
+
+const logEventsLogDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.events.log')) {
+    log(
+      '"logging.events.log" has been deprecated and will be removed ' +
+        'in 8.0. Moving forward, log levels can be customized on a per-logger basis using the new logging configuration. '
+    );
+  }
+  return settings;
+};
+
+const logEventsErrorDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.events.error')) {
+    log(
+      '"logging.events.error" has been deprecated and will be removed ' +
+        'in 8.0. Moving forward, you can use "logging.root.level: error" in your logging configuration. '
```

Comment:
super nit: Could still link to the docs for both of the `logging.events` deprecations

Reply:
@lukeelmers is it ok if we do that once the docs for the KP logging system are added and I go back and change the other links?

Reply:
works for me 🙂